src/aap_eda/settings/default.py

```diff
@@ -176,10 +180,15 @@ def _get_secret_key() -> str:
         "USER": settings.get("DB_USER", "postgres"),
         "PASSWORD": settings.get("DB_PASSWORD"),
         "NAME": settings.get("DB_NAME", "eda"),
+        "OPTIONS": {
+            "sslmode": settings.get("EDA_PGSSLMODE", default="allow"),
```

Comment:
EDA prefix is automatically added by dynaconf. This must be just `PGSSLMODE` same for the rest